src/spyglass/common/common_behav.py

```diff
@@ -214,3 +217,38 @@ def make(self, key):
 
     def fetch_nwb(self, *attrs, **kwargs):
         return fetch_nwb(self, (Nwbfile, "nwb_file_abs_path"), *attrs, **kwargs)
+
+    @classmethod
+    def get_abs_path(cls, key: Dict):
+        """Return the absolute path for a stored video file given a key with the nwb_file_name and epoch number
+
+        The SPYGLASS_BASE_DIR environment variable must be set.
+
+        Parameters
+        ----------
+        key : dict
+            dictionary with nwb_file_name and epoch as keys
+
+        Returns
+        -------
+        nwb_video_file_abspath : str
+            The absolute path for the given file name.
+        """
+        base_dir = pathlib.Path(os.getenv("SPYGLASS_BASE_DIR", None))
+        assert (
+            base_dir is not None
+        ), "You must set SPYGLASS_BASE_DIR environment variable."
+        video_info = (cls & key).fetch1()
+        nwb_path = f"{base_dir}/raw/{video_info['nwb_file_name']}"
```

Comment:
I would get the abs path with `Nwbfile.get_abs_path`; that way you may not need the check in lines 237-240